apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -442,6 +484,10 @@ type ResourcePlacementStatus struct {
 	// Conditions is an array of current observed conditions for ResourcePlacementStatus.
 	// +optional
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+
+	// Overrides contains a list of override snapshot associated with the selected resources.
+	// +optional
+	Overrides []OverrideSnapshotIdentifier `json:"overrides,omitempty"`
```

Comment:
Shall we just separate them to make them more explicit?

ApplicableResourceOverrides []NamespacedName

ApplicableClusterResourceOverrides []string

type NamespacedName struct {
  namespace
  name
}